crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -0,0 +1,798 @@
+use crate::fullscreen_vertex_shader::fullscreen_shader_vertex_state;
+use bevy_app::{App, Plugin};
+use bevy_asset::{load_internal_asset, HandleUntyped};
+use bevy_ecs::{
+    prelude::{Component, Entity},
+    query::{QueryState, With},
+    system::{Commands, Query, Res, ResMut, Resource},
+    world::{FromWorld, World},
+};
+use bevy_math::UVec2;
+use bevy_reflect::{Reflect, TypeUuid};
+use bevy_render::{
+    camera::ExtractedCamera,
+    prelude::Camera,
+    render_graph::{Node, NodeRunError, RenderGraph, RenderGraphContext, SlotInfo, SlotType},
+    render_phase::TrackedRenderPass,
+    render_resource::*,
+    renderer::{RenderContext, RenderDevice, RenderQueue},
+    texture::{CachedTexture, TextureCache},
+    view::ViewTarget,
+    Extract, RenderApp, RenderStage,
+};
+use bevy_utils::HashMap;
```

Comment:
We need to import info_span when tracing is enabled.

```suggestion
#[cfg(feature = "trace")]
use bevy_utils::tracing::info_span;
use bevy_utils::HashMap;
```